UltiSnips/elixir.snippets

```diff
@@ -0,0 +1,126 @@
+snippet if
+if ${1:condition} do
+	${2:expression}
+${3:end}
```

Comment:
I don't like this because it deviates from the standard, and
```
if<tab>true<tab>foo<tab>else<tab>foo
```
doesn't seem any better than
```
if<tab>true<tab>foo<CR>else<CR>foo
```
And when you don't want an else it's actually worse.